slides/01-introduction.qmd

```diff
@@ -141,3 +141,116 @@ inference vs. prediction
 library(tidymodels)
 ```
 
+##  {background-image="images/tm-org.png" background-size="contain"}
+
+## The whole game
+
+```{r diagram-split, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-split.jpg")
+```
+
+## The whole game
+
+```{r diagram-model-1, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-model-1.jpg")
+```
+
+## The whole game
+
+```{r diagram-model-n, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-model-n.jpg")
+```
+
+## The whole game
```

Comment:
Maybe add some notes or make a change to the diagram that reflects the iterative nature of resampling. 

Looking at this, I think that people will think that you are just fitting the model to the training set and moving on.

Reply:
Are you refering to the flow chart in [whole-game-model-n.jpg](https://github.com/tidymodels/workshops/pull/137/files#diff-9feda7e8a16db53708b63fa04797d3766c4eda250f24d4e234d97f976fc286f2)? Or the final version of the flow chart?

For the `whole-game-model-n` version:
My motivation for showing one fit on the entire training set (the chart one prior to this) was that this is what we are doing in the "What makes a model" deck and I'd like to end decks 2-4 with the corresponding stage of this diagram series for the whole game. The intension of this one is to motivate resamples. If it's too tempting to read this as "we are fitting several models straight to the training data", we could skip `model-n` and go straight to `whole-game-resamples`.

For the final version:
Do you mean with iterativeness that the resampling scheme typically involves multiple different splits of the training data? I.e. the "sliced" visualization of the training data seen in some omnigraffle version from previous years?

Reply:
> If it's too tempting to read this as "we are fitting several models straight to the training data"

I think that having some `:::notes` for the speakers to tell the audience that were are not doing a single fit on the entire training set would be enough.